### ISSUE TITLE
nanotts now in arch linux (aur)

### ISSUE THREAD

**Issue by carlocapocasa** [gmn/nanotts#3]
Hi Gregory,

thanks for creatung nanotts. I believe I just made it available for its first linux distribution via the unofficial arch linux aur archive.

https://aur.archlinux.org/packages/nanotts-git/

Carlo

**Comment by gmn** [comment]
Carlo, that's awesome. I have a quick question for you, because you made it available as an installable package -- would it be terrible if I worked on it and improved it?  There were a number of improvements I still wanted to make but got hired somewhere midway through working on it and put it on hold. For instance, it creates a tmp file to the current-working-directory each time a snippet is generated. This is horrible behavior. There's no reason it shouldn't stream directly to the pcm device. I just got lazy because I was running out of time. Best to you, Greg

**Comment by gmn** [comment]
Of course, I meant it implicitly that some of the improvements might involve changing the input flag ordering. Of course it might also result in better --help message and documentation, to aid in using it :)
~Greg

**Comment by carlocapocasa** [comment]
Hi Greg, the package I made ends with "-git". This is an arch convention and arch users generally know that installing or upgrading this package will use the latest development code with no guarantees of any kind. Any changes you may make, including the user interface, are therefore expected and probably simply followed with interest. Arch users can also easily keep or reinstall old versions of their packages and those who install -git packages can be assumed to know how to do so. I hope this helps, and do ask away! Carlo

**Comment by Ma5t3rful** [comment]
@capocasa it's now flagged as out of date. Could you please update it?
